### PR TITLE
3.0 updates

### PR DESCRIPTION
**Pull request by bicknellr**

### CODE REVIEW

**Comment by keanulee** [comment]
Change base branch to __auto....

File: demo/index.html [hunk 1]
```diff
@@ -17,25 +17,22 @@
...
     <script type="module">
-const $_documentContainer = document.createElement('template');
+      import '@polymer/iron-demo-helpers/demo-pages-shared-styles.js';
+      import '@polymer/iron-demo-helpers/demo-snippet.js';
+      import '@polymer/polymer/polymer-legacy.js';
+      import '../iron-image.js';
+
+      window.load = function(id) {
+        document.getElementById(id).src = './polymer.svg?' + Math.random();
+      };
```
Comment (keanulee) [inline demo/index.html]: `type=module` is deferred, so I'm not sure setting `window.load` works (just check, no action needed if it works).

File: demo/index.html [hunk 2]
```diff
@@ -17,25 +17,22 @@
...
+    </script>
 
-$_documentContainer.innerHTML = `<custom-style>
+  </head>
+  <body unresolved>
+    <custom-style>
       <style is="custom-style" include="demo-pages-shared-styles">
```
Comment (keanulee) [inline demo/index.html]: So moving `<style is="custom-style" include="demo-pages-shared-styles">` to `<body>` works? Wished I knew that before keeping template stamping in JS in other elements 😦

Comment (bicknellr) [reply demo/index.html]: I'm actually not sure why it works here, none of the others I've done before this have. :/

File: demo/index.html (hunk 1 above)
Comment (bicknellr) [reply demo/index.html]: This function is called by some inline click listeners on buttons in the page below. I think it'll be ok as long as it runs before the user tries to click them.

**Pull request merged**